web/settings.py

```
@@ -1,12 +1,15 @@
 import os
 import sys
 from pathlib import Path
-
+from dotenv import load_dotenv
 import environ
+import logging
 
 BASE_DIR = Path(__file__).resolve().parent.parent
-
+load_dotenv()
 SECRET_KEY = "django-insecure-5kyff0s@l_##j3jawec5@b%!^^e(j7v)ouj4b7q6kru#o#a)o3"
```

Ruff (0.8.2)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)